carla/models/catalog/catalog.py

```diff
@@ -56,25 +60,27 @@ def __init__(
         else:
             raise Exception("Model type not in catalog")
 
-        self._model = load_model(model_type, data.name, ext, cache, models_home, **kws)
+        self._model: Any = load_model(
+            model_type, data.name, ext, cache, models_home, **kws
+        )
 
-        self._continuous = data.continous
-        self._categoricals = data.categoricals
+        self._continuous: List[str] = data.continous
+        self._categoricals: List[str] = data.categoricals
 
-        self._feature_input_order = feature_input_order
+        self._feature_input_order: List[str] = feature_input_order
```

Comment:
We already defined the type of feature input order parameter in the function signature, not sure if adding it here to the property again is not polluting the code?

Reply:
I thought about taking special caution to properties, but in fact your right.
Will remove them in further commits.